test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Random.Tests.ps1

```diff
@@ -59,14 +59,29 @@ Describe "Get-Random DRT Unit Tests" -Tags "CI" {
     )
 
     # minimum is always set to the actual low end of the range, details refer to closed issue #887.
-    It "get a correct random number for '<Name>'" -TestCases $testData {
+    It "Should return a correct random number for '<Name>'" -TestCases $testData {
         param($maximum, $minimum, $greaterThan, $lessThan, $type)
 
         $result = Get-Random -Maximum $maximum -Minimum $minimum
         $result | Should -BeGreaterThan $greaterThan
         $result | Should -BeLessThan $lessThan
         $result | Should -BeOfType $type
+    }
+
+    It "Should return correct random numbers for '<Name>' with Count specified" -TestCases $testData {
+        param($maximum, $minimum, $greaterThan, $lessThan, $type)
+
+        $result = Get-Random -Maximum $maximum -Minimum $minimum -Count 1
+        $result | Should -BeGreaterThan $greaterThan
+        $result | Should -BeLessThan $lessThan
```

Comment:
Looking asserts above it seems it should be BeLessOrEqual in all tests. What do you think?

Reply:
I agree with you. I've made the modifications.